### PR TITLE
v4: Fix Dart Sass `divide` usage

### LINKED ISSUE #34353: Sass / deprecation for branch 4
Sass / deprecation was fixed in branch 5.
Is the the deprecation expected to be resolved in branch 4 as well?

I know this means switching from libsass to dart, but there are projects that directly import bootstrap4 scss sources

_Originally posted by @castellinosoftware in https://github.com/twbs/bootstrap/issues/34051#issuecomment-868634798_

### PR DESCRIPTION
**Pull request by slavede**
Applied sass-migrator for division deprecation issue. Fixes #34353

### CODE REVIEW

File: scss/vendor/_rfs.scss [hunk 1]
```diff
@@ -43,21 +44,21 @@ $rfs-base-font-size-unit: unit($rfs-base-font-size);
 
 // Remove px-unit from $rfs-base-font-size for calculations
 @if $rfs-base-font-size-unit == "px" {
-  $rfs-base-font-size: $rfs-base-font-size / ($rfs-base-font-size * 0 + 1);
+  $rfs-base-font-size: divide($rfs-base-font-size, $rfs-base-font-size * 0 + 1);
 }
 @else if $rfs-base-font-size-unit == "rem" {
-  $rfs-base-font-size: $rfs-base-font-size / ($rfs-base-font-size * 0 + 1 / $rfs-rem-value);
+  $rfs-base-font-size: divide($rfs-base-font-size, $rfs-base-font-size * 0 + 1 / $rfs-rem-value);
```
Comment (tu4mo) [inline scss/vendor/_rfs.scss:50]: There's a couple of `/` divisions in this file.

Comment (slavede) [reply scss/vendor/_rfs.scss:50]: I see...it seems sass-migrator didn't took nested calculations into consideration, will go manually through those.

**Review by DerZyklop — COMMENTED** [review]
@slavede changed every `/ 2` to `* 0.5`. It would be a no-brainer to at least merge these changes into existing bootstrap v4

**Review by snewer — APPROVED** [review]

File: scss/mixins/_forms.scss [hunk 2]
```diff
@@ -10,6 +10,7 @@
 //
 // Example usage: change the default blue border and shadow to white for better
 // contrast against a dark gray background.
+
```
Comment (ffoodd) [inline scss/mixins/_forms.scss:13]: Unrelated change

File: scss/utilities/_embed.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Credit: Nicolas Gallagher and SUIT CSS.
 
+
```
Comment (ffoodd) [inline scss/utilities/_embed.scss:3]: Unrelated change

File: scss/vendor/_rfs.scss [hunk 4]
```diff
@@ -9,6 +9,7 @@
 // Configuration
```
Comment (ffoodd) [inline scss/vendor/_rfs.scss:9]: RFS is a vendor, please do not edit it directly. It needs to be updated from upstream, see [RFS repository](https://github.com/twbs/rfs).

**Review by ffoodd — CHANGES_REQUESTED** [review]
RFS should be updated in a separate PR I guess, or at least make sure you're using latest RFS instead of modifying it.

Minor a few unrelated line breaks, this looks great: thanks a lot for your effort :heart:

File: scss/vendor/_rfs.scss (hunk 4 above)
Comment (slavede) [reply scss/vendor/_rfs.scss:9]: But it doesn't say inside which exact version of rfs we have here? Should I update the latest? If so, it will pull in a lot more changes to it...this way, it would just fix the warnings...and leave functionality/behavior the same

Comment (ffoodd) [reply scss/vendor/_rfs.scss:9]: Leave RFS for now, we'll upgrade it in another PR. @XhmikosR Does this sound right to you?

**Comment by XhmikosR** [comment]
@ffoodd Yeah, RFS shouldn't be touched for sure.

Generally, we should be very careful with this PR. It still has some issues on main, I wouldn't hurry backporting it unless we are 100% sure it's OK.

**Comment by mdo** [comment]
Fixed linting issues, removed unrelated changes, and removed RFS in #34571.
